themes/default-bootstrap/contact-form.tpl

```diff
@@ -143,6 +143,7 @@
 						<label for="message">{l s='Message'}</label>
 						<textarea class="form-control" id="message" name="message">{if isset($message)}{$message|escape:'html':'UTF-8'|stripslashes}{/if}</textarea>
 					</div>
+					{hook h='displayGDPRConsent' contactForm='contactForm'}
```

Comment:
What are the parameters expected by the hook `displayGDPRConsent`? It can be id_module and now contactForm.

Reply:
@Joukz and I discussed about it yesterday, it's because the GDPR module expects a module ID but this is managed by the Core in 1.6

Reply:
okay, so we must send a module ID here instead?

Reply:
Hey guys,
How is this line supposed to work with psgdpr v1.0.3? I can't see anything able to handle this "contactForm" parameter in the hookDisplayGDPRConsent function.
Thanks a lot.

Reply:
@aeyoll this is for psgdpr for 1.6 not 1.7

Reply:
@kpodemski Yes, I'm working on multiple 1.6 shops, and I was willing to add this features to the contact forms, but I don't understand how could it be working.